src/Components/test/testassets/Components.TestServer/RazorComponents/Pages/Forms/ModifyHttpContextForm.razor

```diff
@@ -27,6 +27,7 @@ else if (_submitted)
     bool _submitted = false;
     bool _submitting = false;
     EditContext _editContext = new EditContext(new object());
+    [Parameter] public string? OperationId { get; set; }
```

Comment:
Could you clarify what all the `OperationId` parameters are used for? I see the parameters are declared but they don't appear to be set or used anywhere.

Reply:
If it's not used, does it work to remove that segment from the `@page` definition? I'm not sure why it would be there if it's not used.

Reply:
It doesn't work if I remove it. I'll try to figure out where exactly it is used and include it.

Reply:
Did you also update `FormsHandlingTest` to update the URLs it uses?